job-server/src/main/scala/spark/jobserver/io/FileCacher.scala

```diff
@@ -50,10 +50,10 @@ trait FileCacher {
       override def accept(dir: File, name: String): Boolean = {
         val prefix = appName + "-"
         if (name.startsWith(prefix)) {
-          val suffix = name.substring(prefix.length)
-          (Pattern findFirstIn suffix).isDefined
```

Comment:
I think it is ensuring that the filename has suffix as date. Why are we removing this logic?